charts/azure-aks-aso/templates/credentials.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     {{- include "capz.commonLabels" . | nindent 4 }}
   annotations:
     helm.sh/resource-policy: keep # credentials have to be available throughout a delete operation
-stringData:
+stringData: # https://azure.github.io/azure-service-operator/guide/aso-controller-settings-options/
```

Comment:
This link describes the global aso-controller-settings Secret but this is a per-resource Secret that only contains credentials and has a different set of known fields, so a better link would be here: https://azure.github.io/azure-service-operator/guide/authentication/credential-format/